lib/client/api.go

```diff
@@ -2541,6 +2559,200 @@ func (tc *TeleportClient) Login(ctx context.Context) (*Key, error) {
 	return key, nil
 }
 
+type pwdlessPrompt struct {
+	Out io.Writer
+}
+
+func (l pwdlessPrompt) PromptPIN() (string, error) {
+	fmt.Fprintln(l.Out, "Enter your security key PIN:")
+	pwd, err := passwordFromConsoleFn()
+	if err != nil {
+		fmt.Fprintln(l.Out, err)
```

Comment:
This looks touch non-standard. What kind of errors can we get here that the user should see? Can we trust the caller to report these to the user instead?

Also, it looks like it goes out to stdout, while some other errors will go to stderr instead?

Reply:
This block is essentially lifted from AskOTP / AskPassword functions. I have a follow up CL that refactors all those functions to a singular ReadPassword function, I'll paste the link here once it's out.

Reply:
See #11346. There's some overlap between that PR and this one, so I'll do a small fix to whichever gets merged last.